dev/com.ibm.ws.security.openidconnect.client/test/com/ibm/ws/security/openidconnect/client/web/OidcRedirectServletTest.java

```diff
@@ -6,7 +6,7 @@
  * http://www.eclipse.org/legal/epl-v10.html
  *
  * Contributors:
- *     IBM Corporation - initial API and implementation
+ * IBM Corporation - initial API and implementation
```

Comment:
@utle please check the copyright dates throughout this PR

Reply:
done